src/index.jsx

```diff
@@ -1,42 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Navigate, BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import { CookiesProvider } from 'react-cookie';
 import './index.css';
 
 // page imports
 import SitesCreateView from './pages/sites-create/sites-create';
 import LoginView from './pages/login/login';
 import LoginResetPasswordView from './pages/login-reset-password/login-reset-password';
-import TeacherView from './pages/master-teachers/master-teachers';
-import TeachersExportDataView from './pages/master-teachers-export-data/master-teachers-export-data';
+// import TeacherView from './pages/master-teachers/master-teachers';
+// import TeachersExportDataView from './pages/master-teachers-export-data/master-teachers-export-data';
 import SettingsView from './pages/settings/settings';
 import SettingsEditView from './pages/settings-edit/settings-edit';
 import AreaManagement from './pages/area-management/area-management';
 import AssessmentScorecardInput from './pages/assessment-scorecard-input/assessment-scorecard-input';
 import AreaDetails from './pages/area-details/area-details';
 import PeopleView from './pages/people/people';
+import NotFoundView from './pages/not-found/not-found';
+import AccessDeniedView from './pages/access-denied/access-denied';
 import EmailAction from './components/EmailAction/EmailAction';
 import ProtectedRoute from './common/ProtectedRoute';
 import { AUTH_ROLES } from './common/config';
 
 const { ADMIN_ROLE, USER_ROLE } = AUTH_ROLES;
-// TODO: add protected routes -- use current few as examples
-// specify component to render if access allowed, redirectPath if access denied
-// use roles from AUTH_ROLES; add to roles list
 
 ReactDOM.render(
   <React.StrictMode>
     <CookiesProvider>
       <Router>
         <Routes>
           <Route path="/" exact element={<LoginView />} />
-          <Route path="/sites/create/" exact render={() => window.location.replace('/sites')} />
-          <Route path="/sites/create/:areaId" exact element={<SitesCreateView />} />
           <Route path="/login" exact element={<LoginView />} />
           <Route path="/login/reset-password" exact element={<LoginResetPasswordView />} />
-          <Route path="/master-teachers" exact element={<TeacherView />} />
-          <Route path="/master-teachers/export-data" exact element={<TeachersExportDataView />} />
+          {/* <Route path="/master-teachers" exact element={<TeacherView />} /> */}
+          {/* <Route path="/master-teachers/export-data" exact element={<TeachersExportDataView />} /> */}
```

Comment:
Not sure if these are going to be used in the future so let me know if I should keep/uncomment or remove them.

Reply:
You can take them out